templates/csharp/src/Appwrite/Models/Rule.cs.twig

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Appwrite
```

Comment:
We can use a twig var here, instead of Appwrite hard-coded, might use it for other stuff in the future